moduleroot/.github/workflows/release.yml.erb

```
@@ -16,10 +16,16 @@ jobs:
     with:
 <%- if @configs['with'] -%>
 <%- @configs['with'].each do |k,v| -%>
-      <%= k %>: <%=  v %>
+<%- if v.is_a?(String) -%>
+      <%= k %>: '<%= v %>'
+<%- else -%>
+      <%= k %>: <%= v %>
 <%- end -%>
 <%- end -%>
+<%- end -%>
+<%- if @configs['with'].nil? || !@configs['with'].has_key?('allowed_owner') -%>
```

This code is indeed intended to prevent running on forks. Isn't this equivalent?
```suggestion
<%- unless @configs['with']&.has_key?('allowed_owner') -%>
```

---

Oh, advanced ruby ...
I'll give it a try :)